src/App.js

```diff
@@ -30,14 +31,14 @@ function App(props) {
             )}
           />
           {/* Wrap AuthProvider round restricted routes so they don't render if user isn't logged in */}
-          <AuthProvider>
+          {/* <AuthProvider> */}
```

Comment:
Need to uncomment this to allow `Redirect to='/Login'/` (in AuthContext.js) to take effect